pkg/apis/db/v1alpha1/cassandracluster_types_test.go

```diff
@@ -476,6 +476,22 @@ func TestSetDefaults(t *testing.T) {
 			Namespace: "default",
 		},
 		Spec: CassandraClusterSpec{
+			Topology: Topology{
+				[]DC{
+					{
+						Name: "dc1",
+					},
+					{
+						Name: "dc2",
+						Resources: CassandraResources{
```

Comment:
I completely forgot that we were using a specific type for Resources parameter at the root level. I think we should use  *v1.ResourceRequirements is possible instead of **CassandraResources**. I really can't remember why we started to create our own type to just do exactly the same. Please see if you can replace that type by just *v1.ResourceRequirements.

Reply:
Modification done on `CassandraCluster` types definition. It now needs to update `zz generrated deepcopy` & `openapi` to match.

I'll also update the definition in `Backup` & `Restore` types Definition